### ISSUE TITLE
Organizational | Non-Profit

### ISSUE THREAD

**Issue by asddsaz** [privacytoolsIO/privacytools.io#899]
I would like to bring up the idea of setting up a legally binding process for privacytools.io to run on. 
If it became a [US non-profit](https://www.wikipedia.org/wiki/501(c)_organization) it would be able to file [Articles of Incorporation](https://www.wikipedia.org/wiki/Articles_of_incorporation) with [Bylaws](https://www.wikipedia.org/wiki/By-law) and a [board of directors](https://www.wikipedia.org/wiki/Board_of_directors).

This way our internal rules could become legally registered:
https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md

It would also help ensure the long term functionality of the privacytools.io community.

**For US Non-Profit Registration**: 

Plenty of online tutorials for how to do this: https://www.usa.gov/start-nonprofit

**Related Links**:

https://www.wikipedia.org/wiki/Nonprofit_organization#By_jurisdiction

_____________________

Obviously, creating a non-profit can cost something so therefore if you wish to support this issue. I recommend putting a [bounty on it](https://www.bountysource.com/issues/73699985-organizational-501-c-non-profit)! We accept bounties via [Bounty Source](https://www.wikipedia.org/wiki/Bountysource).

**Comment by JonahAragon** [comment]
How will this benefit us exactly? Forming a corporate entity is not a guarantee of funding, etc so I'm not sure how it *ensures* the long-term continuation of this project.

**Comment by asddsaz** [comment]
> How will this benefit us exactly 

@JonahAragon Many ways, US Non-profits legally must be very transparent (excluding religious charities).

With the release of privacytools.io services transparency is very important.

> Forming a corporate entity is not a guarantee of funding, etc so I'm not sure how it ensures the long-term continuation of this project.

Crafting a legal entity can help set up a legal process in case the owner/chair/president becomes deceased. It can also enable privacytools.io to apply for certain grants that they may not have otherwise had access to.

**Comment by five-c-d** [comment]
> How will this benefit us exactly?

Right now you are accepting donations without registering as a non-profit, which means you are supposed to report the donation-income as taxable.  But more crucially, the donor must report the expenditure, as an expense.  If you register as a 501.c.4 in the USA, you can become an educational foundation, and accept donations which are tax-deductible.  You can also accept dark money, from all the privacy-conscious individuals that want to give, but don't want to reveal their names in the transparency-reports.  Money that privacyToolsIO **spends** would be revealed, but the names of donors, would not.  

So the answer is, TLDR:  if you are a 501.c.4 or 501.c.3 or whatever, you can GET more donation-bucks, and you can also usually KEEP more donation-bucks.  Merely being a proper foundation, also tends to open doors:  major donors won't send paypal of ten thousand bucks to some github-username, but they *might* write a cheque made out to "The www.PrivacyTools.IO Foundation, POBox 12345, Dover, Delaware" with a little place in the memo-field saying 'you can list our name in your /supporters portion of the website' or alternatively 'we prefer to remain anonymous' 

>  How does this cost us?  

It is a huge timesink to set up properly.  You also almost certainly need a lawyer, to make sure you aren't screwing something up.  Somebody will have to make sure all the paperwork is correct.  Somebody will have to file the proper disclosures, every year AT MINIMUM, and if you start getting involved in political lobbying or advocacy (global warming was mentioned once as being "something privacyToolsIO project-leads think is related to the mission") you will have to be VERY careful not to violate any federal laws related to campaigning.  

The main downside is opportunity cost:  every minute you spend messing around with 501.c.4 websites that explain the (exhaustingly complex) process of becoming one, without footgunning yourself or any of the other privacyToolsIO project-leads, *especially* if some or most of you wish to remain anonymous rather than listing your home address and full legal name, is a minute you could not devote to hardware-sections, software-tools, and things like sleeping/eating/hygiene/work/etc  :-)

**Comment by StopTrackingUs** [comment]
Have you considered becoming an autonomus arm of another user liberation/privacy non-profit?  It would save 100% of lawyer fees and your donations would be covered back to the date the non-profit originally got their status.

**Comment by five-c-d** [comment]
No need to become an autonomous arm of some organization, privacyToolsIO can reap 90% of the tax benefits with a one-paragraph contractual arrangement -- just, make a deal with some existing privacy-organization, to accept earmarked donations on behalf of privacyToolsIO, using **their** 501c4 or 501c3 or UkCharity or BelgiumNGO or whatever legal status, and their payment processor, and their banking arrangements, and so on.  

p.s.  That is a genius idea  :-)  Wish I had thought of it!  Nice work @StopTrackingUs   This is what signalapp has been doing for some years now, they don't take donations directly, Snowden's FreedomOfThePress Foundation has a place where Signal-earmarked donations can be accepted.  I don't know the details of how the money is passed along, but I presume that privacyToolsIO can just send the server-bills to the sugar-daddy-501c4 entity, and the money collected in the earmarked-fund will be paid.  

N.B. that you have to ***really trust*** the sugar daddy, since they *WILL* have control of the donated monies utterly ('earmark' is not a legal thing!)  But as long as you pick a trustworthy sugar-daddy to handle privacyToolsIO donations, or a portion thereof, you would avoid ~90% of the timesink-problem, and yet still benefit from ~90% of the expected percentage-increases in donation-flow.  You can keep accepting donations the current way (paypal/etc) if you like, or you can even get multiple sugar-daddy organizations to expand the number of potential donors that hear about privacyToolsIO.   

I will see if I can put together a list of prospective sugar-daddy non-profits that might consider making a page on their webserver where earmarked privacyToolsIO donations are accepted.  In addition to Snowden's organization, just off the top of my head EFF, TorProject, MozillaFoundation ... there are also a lot of places like NOYB, PI, EPIC, CDT, EDRi, ACLU maybe... even if only one of them is willing to help support privacyToolsIO by processing donations, that would be helpful to offload the lawyer&accounting paperwork pain.  But why not half a dozen of them?  :-)   Never hurts to have the privacyToolsIO brand linked to existing places which have a good reputation of their own, and are aligned with the mission of beating mass surveillance in the digital realm.

**Comment by StopTrackingUs** [comment]
We are not a big name in the industry, but we do have 501c3 status so if any of the big names turn the project down we would be open to assist.  Is there a reason why privacytools does not run it's own servers or are server cost related to co-location?

**Comment by Mikaela** [comment]
Do Non-Profits exist in other countries or would applying for it make sense outside of the USA?

In Finland an association would be relatively simple to begin, but collecting donations is not. [Federated Networks Association](https://feneas.org) is [thinking about how to do it](https://git.feneas.org/feneas/association/issues/55).

It might also be worth to take a look at [the other Librehosters and if they are legal entities](https://libreho.st/).

**Comment by five-c-d** [comment]
> Do Non-Profits exist in other countries

Yes, there are UK-based charities like Privacy International, and for instance [EDRi](https://en.wikipedia.org/wiki/European_Digital_Rights#Members) member-groups -- as well as NYOB methinks -- are various kinds of EU-based not-for-profit entities.  

> would applying for it make sense outside of the USA?

If you are becoming your **own** tax-entity, then choice of jurisdiction really matters.  If you are negotiating an earmark-deal with existing ones, then it still *matters* but mostly from the donor's perspective.  

> collecting donations is not

From the privacyToolsIO perspective, you will want at least one USA-based sugar-daddy that accepts earmarked-donations on your behalf from donors who want tax-related or donor-privacy-related advantages in USD / North America, and you will want in parallel at least one EU-based sugar-daddy that accepts earmarked-donations on your behalf from donors who want tax-related or donor-privacy-related advantages in Euro / GBP / European areas.  

Or you might be able to get away with just one, especially if they have an international donor-network (local chapters or whatever), and a multi-tier 501c3+501c4 arrangement in the USA such as places like the [ACLU:SPT](https://en.wikipedia.org/wiki/American_Civil_Liberties_Union#cite_ref-62) can afford to setup.  Not sure about Asia/Pacific, and whether donors in those regions are going to need non-USA non-EU somethings?

**Comment by blacklight447-ptio** [comment]
Maybe we could make use of this?
https://github.blog/2019-05-23-announcing-github-sponsors-a-new-way-to-contribute-to-open-source/

**Comment by JonahAragon** [comment]
@StopTrackingUs define "own servers", we run all our infrastructure on _dedicated_ servers from OVH. Who are you affiliated with? Feel free to send me an email: `jonah@privacytools.io`.

@blacklight447-ptio cool, joined the waitlist. I'm also going to investigate `FUNDING.yml` for this repo...

**Comment by JonahAragon** [comment]
I'm looking into [Open Collective](https://opencollective.com/) and it looks like taking donations through there and tracking expenses would be pretty beneficial for us. We'd still have to partner with a (preferably non profit) organization to manage finances but the site facilitates that kind of thing apparently.

**Comment by five-c-d** [comment]
I have seen one of the projects that is housed on OpenCollective, it has good transparency when it comes to expenditures -- possibly a little TOO good for a privacy-oriented website run by privacy-oriented people  :-)   https://opencollective.com/wikijs#budget  But it is pretty cool if you are okay with what level of detail it reveals

**Comment by gjhklfdsa** [comment]
> Do Non-Profits exist in other countries or would applying for it make sense outside of the USA?
> 
> In Finland an association would be relatively simple to begin, but collecting donations is not. [Federated Networks Association](https://feneas.org) is [thinking about how to do it](https://git.feneas.org/feneas/association/issues/55).
> 
> It might also be worth to take a look at [the other Librehosters and if they are legal entities](https://libreho.st/).

One possibility is joining/starting organizations in different countries.
Some organizations will also partner with existing non-profits.

LibreOffice does this, their foundation is founded in Germany.
US donors can donate through [SPI](https://www.wikipedia.org/wiki/Software_in_the_Public_Interest).[1](https://www.spi-inc.org/projects/)

Non-profit laws for creating bylaws and what-not vary.
https://www.wikipedia.org/wiki/Nonprofit_organization#By_jurisdiction

**Comment by JonahAragon** [comment]
Looking at this again I think this is kind of out of scope for the project. It would add a lot of complexities we don't have the resources to handle at the moment.

**Comment by JonahAragon** [comment]
Now that the Open Collective Foundation 501(c)(3) has been founded, we should actually try to apply with them.
